plugins/bcm57xx/fu-bcm57xx-common.h

```diff
@@ -0,0 +1,144 @@
+/*
+ * Copyright (C) 2018 Richard Hughes <richard@hughsie.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1+
+ */
+
+#pragma once
+
+#include <glib.h>
+
+#define BCM_VENDOR_BROADCOM			0x14E4
+#define BCM_FIRMWARE_SIZE			0x80000
```

Comment:
This is true for the Dell NIC, however the Talos II and Balckbird have a size of 0x40000

Reply:
Ohh! I assumed they'd all use the same SPI chip, apologies. I've set the default to 0x40000 and quirked the Dell card to 0x80000 now.